test/e2e/testsuites/dynamically_provisioned_volume_replicas_across_zones.go

```diff
@@ -0,0 +1,117 @@
+package testsuites
+
+import (
+	"context"
+
+	"github.com/onsi/ginkgo"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/test/e2e/framework"
+	azDiskClientSet "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/client/clientset/versioned"
+	consts "sigs.k8s.io/azuredisk-csi-driver/pkg/azureconstants"
+	"sigs.k8s.io/azuredisk-csi-driver/pkg/azureutils"
+	testconsts "sigs.k8s.io/azuredisk-csi-driver/test/const"
+	"sigs.k8s.io/azuredisk-csi-driver/test/e2e/driver"
+	testtypes "sigs.k8s.io/azuredisk-csi-driver/test/types"
+)
+
+type DynamicallyProvisionedVolumeReplicasAcrossZones struct {
+	Pod                    testtypes.PodDetails
+	CSIDriver              driver.DynamicPVTestDriver
+	Volume                 testtypes.VolumeDetails
+	AzDiskClient           *azDiskClientSet.Clientset
+	StorageClassParameters map[string]string
+}
+
+func (t *DynamicallyProvisionedVolumeReplicasAcrossZones) Run(client clientset.Interface, namespace *v1.Namespace, schedulerName string) {
+	_, maxMountReplicaCount := azureutils.GetMaxSharesAndMaxMountReplicaCount(t.StorageClassParameters, false)
+	// create the storageClass
+	tsc, tscCleanup := t.Pod.Volumes[0].CreateStorageClass(client, namespace, t.CSIDriver, t.StorageClassParameters)
```

Comment:
Why call this explicitly here when `SetupWithDynamicVolumes` will do it for you?